### ISSUE TITLE
Feature Request: Show assigned user of machine in the Device View

### ISSUE THREAD

**Issue by nikp123** [gurucomputing/headscale-ui#119]
Hello, if it doesn't go against any design issues/opinions I'd like to ask for a slight change in the device view.

Background:
I share my headscale instance with my family and I'd like for them to be able to reference their own devices with generic device names like ``phone``, ``tablet``, ``laptop``, ``desktop``.... etc. This isn't a problem in general because usernames are required for other peoples devices, like ``laptop.nikp123.example.com`` (as an example).

However there's a slight UX issue/annoyance when I go into device view. I cannot correctly determine which users device it is based on the device name alone as the device names are similar/same between users.

I'm asking for the ability to determine the user of the device *without* clicking on the device's name for a guess.

For example I have multiple ideas.
We could do show the FQDN of a device like ``laptop.nik.example.com`` or just add a separator and then the users name, like so:
![image](https://github.com/gurucomputing/headscale-ui/assets/4696350/95fbc0c5-1127-42a7-9a5a-a01f15a614c2)

I really have no preference over the system chosen, but I believe choosing any of the two would be a massive quality of life improvement.

**Comment by routerino** [comment]
The search filter will already accept usernames, but agreed that feature is relatively hidden. Adding the username should be fine, just need to test how that affects longer device/usernames.

**Comment by nikp123** [comment]
> The search filter will already accept usernames, but agreed that feature is relatively hidden. Adding the username should be fine, just need to test how that affects longer device/usernames.

Yeah, it'd be valid to test that out first before my change annoys other users.
